packages/features/form-builder/FormBuilder.tsx

```diff
@@ -45,6 +44,13 @@ function getCurrentFieldType(fieldForm: UseFormReturn<RhfFormField>) {
   return fieldTypesConfigMap[fieldForm.watch("type") || "text"];
 }
 
+function isRequiredField(field: RhfFormField) {
+  // 'location' must be shown as required in the UI(but at backend it is not required because not specifying it defaults to CalVideo)
+  // So, handle it in UI only instead of updating bookingFields config in getBookingFields
+  // TODO: FormBuilder must not be made aware of BookingFields, so move this "location" check to outside FormBuilder where the component is used.
+  return field.name === "location" ? true : field.required;
```

Comment:
This makes a new change as well where "location" field is shown as required now. This created confusion earlier for some users.